lib/galaxy/model/__init__.py

```diff
@@ -215,6 +216,20 @@ class _HasTable:
     _HasTable = object
 
 
+# Naming convention applied to database constraints and indexes.
+# All except "ix" are conventions used by PostgreSQL by default.
+# We keep the "ix" template consistent with historical Galaxy usage.
+#
+# NOTE: If editing, also update model.migrations.utils.DbObjectNames.
+NAMING_CONVENTION = {
+    "pk": "%(table_name)s_pkey",
+    "fk": "%(table_name)s_%(column_0_name)s_fkey",
+    "uq": "%(table_name)s_%(column_0_name)s_key",
+    "ck": "%(table_name)s_%(column_0_name)s_check",
+    "ix": "ix_%(table_name)s_%(column_0_name)s",
```

Comment:
How does this work for composite indexes ?

Reply:
It doesn't: it's only used for autogenerated indexes on a single column. Composite indexes require the `Index` construct which requires a name argument [(ref)](https://docs.sqlalchemy.org/en/14/core/constraints.html#indexes). 

However, now that you mention it, it would make sense to use the naming helper `DbObjectNames.index()` in that case. For that, I need to move DbObjectNames out of migration utilities into a separate module (in `galaxy/model`). I'll also modify that method so it can optionally accept a list of columns. If this makes sense, I'll make the change. (we could do the same for composite foreign keys, but in that case I'd wait until we need it.)

Reply:
sounds good to me.